backend/service/authn/oidc.go

```diff
@@ -451,11 +453,33 @@ func (cc *ClaimsConfig) ClaimsFromOIDCToken(ctx context.Context, t *oidc.IDToken
 	if subject == "" {
 		return nil, fmt.Errorf("claims field %s is empty", cc.subjectClaimName)
 	}
+	var groups []string
+	if cc.groupsClaimName != "" {
+		groupsInt, ok := claims["groups"]
```

Comment:
```suggestion
		groupsInt, ok := claims[cc.groupsClaimName]
```

and let's add a test to catch this

Reply:
Removed the hardcoded `"groups"` and added a test that will catch that.